packages/terra-tabs/package.json

```diff
@@ -37,7 +37,10 @@
     "terra-responsive-element": "^5.0.0",
     "terra-theme-context": "^1.8.0",
     "terra-visually-hidden-text": "^2.36.0",
-    "uuid": "7.0.3"
+    "uuid": "7.0.3",
```

Comment:
since uuid is having some issues can we replace this with React [useID()](https://react.dev/reference/react/useId)

Reply:
uuid has been locked to "7.0.3" from "8.2.0" since yesterday's release.

Reply:
Yes, we can remove the UUID from the components list and use the react's useID() instead